test/generators/extra_pages_generator_test.rb

```diff
@@ -0,0 +1,17 @@
+require 'pry'
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.